Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,55 @@
+5. A major goal of the project is to allow for synchronous operations of at least 3 air quality sensors.  Sensors may be either analog or digital, so there must be support for both.  The board must have enough Input/Output (I/O) pins to operate any combination of 3 sensors whether they be digital or analog.   Different sensors interface with a potentially different number of pins.  Preliminary research shows that a sensor may take up to 5 pins to operate.  Other sensors may have more, but it is more common for 5 and under.  This means the board must have at least 15 digital input pins and 15 analog input pins.
+6. The cpu clock speed does not need to be very fast.  The code will be mostly Input/Output heavy with less computation.  This means that there will be a lot of waiting for responses and does not require a high-speed processor.  A cpu clock speed of at least 1 MHz is preferred for fast bootup.  
+7. There must be sufficient memory for the code.  It is unknown how much memory the code will occupy.  Operating code with libraries and compilation could be upwards of or slightly over 150 KB in Flash/Disk/Storage..  To account for the worst case, the board will need over 150 KB.
+8. A huge power-draw would mean that there needs to be a bigger power subsystem which could increase the cost to unacceptable levels.  The power requirements of the board need to be reasonably under or near 240 mW.  Here, we take a voltage supply of 12 V and current of 20 mA to get 240 mW.  12 V with 20 mA is from the power subsystem’s output to provide an adequate operation time or lifespan and  not damaged components.
+9. Some form of battery monitoring capabilities need to be included or applicable.  Telling the user how much longer the device may operate is a must.  This can be done using an additional component or algorithmic measurement.  The device will need to keep track of battery life to prevent data loss by saving and ceasing operations when reaching critical levels.  Doing this will ensure data protection.  
+10. The board itself cannot be too big.  “Too big” here is defined as having a volume greater than 50 cubic inches.  This size was set to prevent the device from proving too large.  50 cubic inches would be about the size of a small novella/book.  .  
+11. The manufacturer needs to be a reputable and safe choice.  Choosing a bad manufacturer could hurt the project.  A stable choice will lower the risk of receiving a malfunctioning board or running out of the specified board supply.  This device is planned to be open to public use and construction, so it will need to be trustworthy.
+12. The board will need to have coding language support for assembly, C, and or Python.  It is expected that the users will have some degree of coding experience and may potentially personalize or improve upon the code. These languages are chosen due to popularity and performance.  C, in particular, is predicted to be very desirable for this project.
```

Comment:
Assembly is not hobbyist-friendly so that should be reconsidered. Be specific about the "and or" - is it OK for just C-only or Python-only or must both C and Python be supported?